meta.go

```diff
@@ -442,8 +443,8 @@ func (u *User) Decode(val interface{}) (err error) {
 
 // License represents a legal licensing agreement
 type License struct {
```

Comment:
Ok if this is messing up the frontend I'm onboard with switching to just an object and no string representation funny business, let's make the switch suggested in this PR!

Just to note original the intention behind this code: at the time I wanted users to be able to directly input  say "GPL-3.0" in the license field (as a string), and have it work. I now agree that's it's too edge-case-ie a scenario to warrent all this code.

While we're here, I'd like to note what github's "license" field in their API looks like for _reference_. I'm not saying we should do this.
```
{
  "key": "gpl-3.0",
  "name": "GNU General Public License v3.0",
  "spdx_id": "GPL-3.0",
  "url": "https://api.github.com/licenses/gpl-3.0",
  "node_id": "MDc6TGljZW5zZTk="
}
```
things worth investigating later: what's spdx? sounds helpful.